poker/src/pages/MainPage/MainPage.jsx

```diff
@@ -1,18 +1,42 @@
 import React from 'react';
 import m from './MainPage.module.scss';
 import MainPageTitle from '../../components/MainPageTitle/MainPageTitle';
+import BlueButton from '../../components/BlueButton/BlueButton';
+import InputComponent from '../../components/InputComponent/InputComponent';
 
-const MainPage = () => (
-  <div className={m.MainPage_wrap}>
-    <MainPageTitle />
-    <h3>Start your planning:</h3>
-    <label>Create session:</label>
-    <button>Start new game</button>
-    <h3>OR:</h3>
-    <label>Connect to lobby by URL:</label>
-    <input />
-    <button>Connect</button>
-  </div>
-);
+const MainPage = () => {
+  const handleClick1 = () => {
```

Comment:
Bad naming, use something like handleStartButtonClick

Reply:
rename to handleCreateNewGame